src/User32.Desktop/User32+KEYBDINPUT.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.InteropServices;
+
+    /// <content>
+    /// Contains the <see cref="KEYBDINPUT"/> nested type.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Contains information about a simulated keyboard event.
+        /// </summary>
+        [StructLayout(LayoutKind.Sequential)]
+        [OfferIntPtrPropertyAccessors]
```

Comment:
You'll know if it works if when the file is open, you can open up your member dropdown bar at the top of your code editor window and see the code generated members listed (in gray color).
What's missing is the `MSBuild:GenerateCodeFromAttributes` value on the file's Custom Tool field in the properties window.